recipe/meta.yaml

```
@@ -28,12 +27,15 @@ outputs:
       run:
         - python
         - googleapis-common-protos >=1.56.2,<2.0dev
-        - protobuf >=3.20.1
-        - google-auth >=1.25.0,<3.0dev
+        - protobuf >=3.19.5,<6.0.0.dev0,!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5
+        - proto-plus >=1.22.3,<2.0.0dev
+        - google-auth >=2.14.1,<3.0.dev0
         - requests >=2.18.0,<3.0.0dev
       run_constrained:
-        - grpcio >=1.33.2,<2.0dev
-        - grpcio-status >=1.33.2,<2.0dev
+        - grpcio >=1.33.2,<2.0dev # [py<311]
+        - grpcio >=1.49.1,<2.0dev # [py>=311]
+        - grpcio-status >=1.33.2,<2.0dev # [py<311]
+        - grpcio-status >=1.49.1,<2.0dev # [py>=311]
         - grpcio-gcp >=0.2.2,<1.0dev
```

```suggestion
        - grpcio-status >=1.33.2,<2.0dev0 # [py<311]
        - grpcio-status >=1.49.1,<2.0dev0 # [py>=311]
        - grpcio-gcp >=0.2.2,<1.0dev0
```
https://github.com/googleapis/python-api-core/blob/fef936315f82a6b1db0bf23595e7bf71c7b503e4/setup.py#L42-L46

---

addressed, thanks